Mortgage-loan-calculator/src/app/admin-login/services/auth.service.ts

```diff
@@ -1,30 +0,0 @@
-import { Injectable } from '@angular/core';
-import { HttpClient, HttpHeaders } from '@angular/common/http';
-import { Observable } from 'rxjs';
-import {FormControl} from "@angular/forms";
-
-const AUTH_API = 'https://mortgage-loan-calculator-front-end2.onrender.com/auth/';
```

Comment:
Explore how values could be provided using InjectionTokens by configuring them on app module.